pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -42,6 +42,7 @@
                 <table id="questionsTable" class="table table-sm table-hover tablesorter">
                     <thead>
                         <tr>
+                            <th width="50">Thumbnail</th>
```

Comment:
Why are we setting an explicit width here? It seems too wide, and it also forces the column to take up a lot of space even when there aren't any thumbnails.

Here we can see that the column looks too wide:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/875833/80308705-fe077580-8795-11ea-9d63-cbc4f3e23c5a.png">

Here is an example where there aren't any thumbnails:

<img width="579" alt="image" src="https://user-images.githubusercontent.com/875833/80308728-1c6d7100-8796-11ea-870f-466e9b21be0d.png">